fe/fe-core/src/main/java/com/starrocks/catalog/mv/MVTimelinessListPartitionArbiter.java

```diff
@@ -56,19 +59,47 @@ public MvUpdateInfo getMVTimelinessUpdateInfoInChecked() throws AnalysisExceptio
 
```

Comment:
If mv's unpartitioned, it will use `MVTimelinessNonPartitionArbiter` to `getMVTimelinessUpdateInfoInChecked` and will not check `partition configuration changed` instead.
```
    public static MVTimelinessArbiter buildMVTimelinessArbiter(MaterializedView mv,
                                                               boolean isQueryRewrite) {
        PartitionInfo partitionInfo = mv.getPartitionInfo();
        if (partitionInfo.isUnPartitioned()) {
            return new MVTimelinessNonPartitionArbiter(mv, isQueryRewrite);
        } else if (partitionInfo.isRangePartition()) {
            return new MVTimelinessRangePartitionArbiter(mv, isQueryRewrite);
        } else if (partitionInfo.isListPartition()) {
            return new MVTimelinessListPartitionArbiter(mv, isQueryRewrite);
        } else {
            throw UnsupportedException.unsupportedException("unsupported partition info type:" +
                    partitionInfo.getClass().getName());
        }
    }
```